config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -361,3 +361,45 @@ periodics:
       - --test-cmd-name=ClusterLoaderV2
       - --timeout=40m
       - --use-logexporter
+
+# cluster volume load scaling by number of nodes
+- name: ci-kubernetes-storage-scalability-cluster-load-scale-by-node
+  tags:
+    - "perfDashPrefix: storage-cluster-load-scale-persistent-volume-by-node"
+    - "perfDashJobType: storage"
+  interval: 1h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-node: "true"
+  annotations:
+    testgrid-dashboards: sig-scalability-experiments
+    testgrid-tab-name: cluster-load-scale-persistent-volume-by-node
```

Comment:
AFAIK no need to do additional thing for test grid tabs, it will be added automatically, but I'm not sure about performance dashboard side.

Reply:
thanks!